jdaviz/core/loaders/resolvers/resolver.py

```diff
@@ -273,6 +364,21 @@ def __init__(self, *args, **kwargs):
         self.file_table.multiselect = False
         self.file_table._selected_rows_changed_callback = self.on_file_select_changed
 
+        # Setup footprint selection
+        if self.app is not None:
+            self.is_wcs_linked = getattr(self.app, '_align_by', None) == 'wcs'
```

Comment:
we probably should also listen to a change in align_by while the loader is opened